restx-core-shell/src/main/java/restx/core/shell/AppShellCommand.java

```diff
@@ -611,6 +611,18 @@ public void run(final RestxShell shell) throws Exception {
                 shell.cd(standardCachedAppPath(appNameArg.get()));
             }
 
+            if(!ModuleDescriptorType.RESTX.resolveDescriptorFile(shell.currentLocation()).exists()) {
+                shell.cd(shell.currentLocation().resolve("srv"));
```

Comment:
Before doing the `cd`, I would check the existence of the `srv` directory: currently if it doesn't exist, it will fail, but the shell will still have the updated current location which doesn't even exist...

Reply:
Done

I ensured to come back to initial shell location after `app run` command